code/__DEFINES/access.dm

```diff
@@ -824,6 +824,8 @@
 	id = ACCESS_LONE_SPACER
 	desc = "Independent Skiff Access"
 
+// for Odyssey scenarios
```

Comment:
remove this comment, accesses are taken on a first available basis, we don't reserve numbers for things

Reply:
I will remove this comment, but my thought process behind it was that there's a comment of "Misc" after the horizon areas end, and the third party ships start.